<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReflectorPropertiesDialog</class>
 <widget class="QDialog" name="ReflectorPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>445</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Modifica il riflettore</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox{background: none}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Poprietà</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1" colspan="3">
       <widget class="QDoubleSpinBox" name="reflectionSpinBox">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox{color: #000000}</string>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="laserPhaseLabel">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="positioningLabel">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="actualPositioningLabel">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>θ&lt;sub&gt;L&lt;/sub&gt;  [gradi]</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="ZRLcheckBox">
        <property name="text">
         <string>ZRL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLabel" name="reflectorKindLabel">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>Descrizione</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>τ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QTextEdit" name="descriptionTextEdit">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit {background-color: #fafafa}
QTextEdit {color: #000000}
QTextEdit {border: 0px solid grey}
QTextEdit {border-radius: 8px}
QTextEdit {padding: 3px}
QTextEdit {margin-right: 10px}</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>Tipo</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>&amp;psi;&lt;sub&gt;R&lt;/sub&gt; [gradi]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>&amp;psi; &lt;sub&gt;&lt;/sub&gt; [gradi]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QSlider" name="positioningSlider">
        <property name="minimum">
         <number>-90</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="maxDeflectionLabel">
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #fafafa}
QLabel {color: #000000}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-right: 10px}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Posizione</string>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-left: 10px}</string>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="xSpinBox">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox{color: #000000}</string>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {background-color: #00c800}
QLabel {color: #fafafa}
QLabel {border: 0px solid grey}
QLabel {border-radius: 8px}
QLabel {padding: 3px}
QLabel {margin-left: 10px}</string>
        </property>
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ySpinBox">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox{color: #000000}</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReflectorPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReflectorPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
